config/initializers/activity.rb

```diff
@@ -40,21 +42,25 @@
                                  default: false
   end
 
-  OpenProject::ProjectActivity.register on: 'WorkPackage',
-                                        attribute: :updated_at
+  OpenProject::ProjectLatestActivity.register on: 'WorkPackage',
+                                              attribute: :updated_at
 
-  OpenProject::ProjectActivity.register on: 'News',
-                                        attribute: :updated_at
+  OpenProject::ProjectLatestActivity.register on: 'Project',
+                                              attribute: :updated_at,
+                                              project_id_attribute: :id
 
-  OpenProject::ProjectActivity.register on: 'Changeset',
-                                        chain: 'Repository',
-                                        attribute: :committed_on
+  OpenProject::ProjectLatestActivity.register on: 'Changeset',
+                                              chain: 'Repository',
+                                              attribute: :committed_on
 
-  OpenProject::ProjectActivity.register on: 'WikiContent',
-                                        chain: %w(Wiki WikiPage),
-                                        attribute: :updated_at
+  OpenProject::ProjectLatestActivity.register on: 'News',
+                                              attribute: :updated_at
 
-  OpenProject::ProjectActivity.register on: 'Message',
-                                        chain: 'Forum',
-                                        attribute: :updated_at
+  OpenProject::ProjectLatestActivity.register on: 'WikiContent',
+                                              chain: %w(Wiki WikiPage),
+                                              attribute: :updated_at
+
+  OpenProject::ProjectLatestActivity.register on: 'Message',
+                                              chain: 'Forum',
+                                              attribute: :updated_at
```

Comment:
Not related to this PR but it might be an idea to by now use `updated_at` as the default. That should have been done when all those `updated_on` columns were renamed to `updated_at`. But definitely can be done in a separate step.

Reply:
This is not possible yet: [`Changeset` is using column `committed_on`](https://github.com/opf/openproject/blob/a9ee223e28bf955ec76872136dc096e8a14e9ee4/config/initializers/activity.rb#L49-L51).

Reply:
Done in #11794